docs/source/gettingstarted.rst

```diff
@@ -221,15 +216,9 @@ A few examples of using this argument::
     # prominence factor of min=1 max=3
     write_xlsx(r, prominence_factors=(1,3))
 
-    # invalid prominence factors min=3
-    write_xlsx(r, prominence_factors=(3,))
-
     # prominence factors of min=20 max=30
```

Comment:
Also need to update this lines and the other ones like it since it's not actually a min and max, it's one value for peaks and one for valleys